### PR TITLE
Keycloak users should not need to understand the depths of Quarkus co…

### LINKED ISSUE #27121: Remove references to Quarkus docs and absolute URLs from HA Guide docs
### Description

This is necessary for clean downstream docs. And we don't want Keycloak users to understand the depths of Quarkus configuration to implement Keycloak HA

### PR DESCRIPTION
**Pull request by ahus1**
Keycloak users should not need to understand the depths of Quarkus cnfiguration to implement Keycloak HA

Closes #27121

### CODE REVIEW

File: docs/guides/high-availability/concepts-threads.adoc [hunk 1]
```diff
@@ -16,13 +16,9 @@ For a configuration where this is applied, visit <@links.ha id="deploy-keycloak-
 
 === Quarkus executor pool
 
-{project_name} requests, as well as blocking probes, are handled by the Quarkus executor pool.
-
-The Quarkus executor thread pool is configured in https://quarkus.io/guides/all-config#quarkus-core_quarkus.thread-pool.max-threads[`quarkus.thread-pool.max-threads`] and has a maximum size of at least 200 threads.
-Depending on the available CPU cores, it can grow even larger.
+{project_name} requests, as well as blocking probes, are handled an executor pool. Depending on the available CPU cores, it has a maximum size of 200 or more threads.
```
Comment (sschu) [inline docs/guides/high-availability/concepts-threads.adoc:19]: in/by an executor pool?

Comment (ahus1) [reply docs/guides/high-availability/concepts-threads.adoc]: Thanks, updated.

**Review by sschu — APPROVED** [review]
LGTM.

**Comment by ahus1** [comment]
@mhajas - approve on @sschu's review and merge when you have the time. Thanks!

**Review by mhajas — APPROVED** [review]
Thank you @ahus1, @sschu!

**Pull request merged**